### DISCUSSION TITLE
数字(%)の不一致

### DISCUSSION THREAD

**Discussion by momocus** [lapras-inc/lapras-user-community#26] [スコアについてのご意見・議論] [answered]
素敵なサービスをありがとうございます。技術力スコアv2を先行体験しました。

気になった点と改善のアイデアを投稿します。

## 気になった点

グラフの上側に表示されている数字%と、下側に表示されている数値%と、目安で表示されている数字%が一致していないように見えます。

![IMG_3557](https://github.com/user-attachments/assets/dcf30be0-df3c-47b3-ad6a-c42fcc5b47f7)

## 改善案

グラフの下側に表示するのは%ではなく3を平均としたスコア（偏差値？）にする方が分かりやすいように思いました。

![IMG_3557](https://github.com/user-attachments/assets/5927762f-1aff-4765-883f-6359f68711e5)

**Comment by nunukim** [comment, marked as answer]
v2へのご意見ありがとうございます！ 

> 目安で表示されている数字%が一致していないように見えます。

目安の数字は以前の v1 における数字でした。
v2の先行体験有効時に非表示にするはずでしたが、SP版に漏れがあったようです 🙏🏻 
ご報告いただきありがとうございました🙇🏻


> グラフの下側に表示するのは%ではなく3を平均としたスコア（偏差値？）にする方が分かりやすいように思いました。

こちらにつきまして、開発チームで議論しましたところ、採用させていただくことになりました！
貴重なご意見ありがとうございました！

**Comment by kawamataryo** [comment]
@momocus 

バグの報告、改善案のご提案ありがとうございました 🙏  
先ほど修正をリリースしました！

<img src="https://github.com/user-attachments/assets/80b219ac-c40b-4225-80f0-189bdea7d10a" width="500">

**Reply by momocus** [reply]
早速のご対応ありがとうございました🙌🙌
